easybuild/framework/easyconfig/types.py

```diff
@@ -76,6 +78,16 @@ def is_value_of_type(value, typ_spec):
                 # check whether all values have allowed types
                 'value_types': lambda val: all([type(el) in extra_reqs['value_types'] for el in val.values()]),
             }
+        elif parent_type == list:
+            extra_req_checkers = {
+                # check required type
+                'element_types': lambda val: all([type(el) in extra_reqs['element_types'] for el in val]),
+            }
+
+        else:
+            raise EasyBuildError("Don't know how to check value with parent type %s", parent_type)
+
+        if extra_req_checkers:
```

Comment:
this check is probably not needed? if `extra_req_checkers` is not defined, something is very wrong